react-components/src/data/details/BasicDetails.js

```diff
@@ -319,7 +324,9 @@ class BasicDetails extends Component {
                         tags={this.state.tags.values}
                         dataSource={this.state.dataSource}
                         onTagClick={this.handleTagClick}
-                        handleTagSelect={this.handleTagSelect}/>
+                        handleTagSelect={this.handleTagSelect}
+                    />
+                    <div>
```

Comment:
Is this extra `div` wrapper required around this `Dialog`?
If not, we should remove it; but if it is required, maybe the `Dialog` should be re-indented inside it.

Reply:
Yup those `<div>` needs to removed. I was trying to figure why `Dialog` was not working. Forgot to remove it once I figured out I need to update `react` and `react-dom`